CHANGES.md

```diff
@@ -10,6 +10,7 @@
 - Add `magik-typed-lint` to check magik typing from CLI/during CI.
 - Fix not indexing `module.def` files.
 - Rename SwChar16VectorEvaluateInvocationCheck to SwChar16VectorEvaluateInvocationTypedCheck.
+- Add NestingLevelCheck to test if a node does not exceed the maximum nesting level.
```

Comment:
I think you should avoid the term "node" here, but rather a method/procedure/body/....